src/target.c

```diff
@@ -90,7 +92,8 @@ stumpless_add_entry( struct stumpless_target *target,
   struct strbuilder *builder;
   size_t builder_length;
   const char *buffer;
```

Comment:
you will need to initialize the buffer pointer to NULL to serve as an indicator (instead of using a separate boolean)